docs/examples/co_simulation_early_return.xml

```diff
@@ -11,6 +11,27 @@
     modelIdentifier="MyLibrary_SpringMassDamper"
     canHandleVariableCommunicationStepSize="true"
     canReturnEarlyAfterIntermediateUpdate="true"/>
+  <UnitDefinitions>
+    <Unit name="rad">
+      <BaseUnit rad="1"/>
+      <DisplayUnit name="deg" factor="57.2957795130823"/>
+    </Unit>
+    <Unit name="rad/s">
+      <BaseUnit s="-1" rad="1"/>
+    </Unit>
+    <Unit name="kg.m2">
+      <BaseUnit kg="1" m="2"/>
+    </Unit>
+    <Unit name="N.m">
+      <BaseUnit kg="1" m="2" s="-2"/>
+    </Unit>
+  </UnitDefinitions>
+  <TypeDefinitions>
+    <Float64Type name="Modelica.SIunits.Inertia" quantity="MomentOfInertia" unit="kg.m2" min="0.0"/>
```

Comment:
Modelica Standard Library version 4.0.0 renamed Modelica.SIunits to Modelica.Units.SI. Is this a real issue for importers?

Reply:
All of the docs/examples that include Modelica named types are "Modelica.SIunits", as are all of the examples in the Standard document itself. It might be wrong, but it's consistent.

Reply:
@beutlich, please make a PR if you want to reflect the changes in Modelica 4.0. From the FMI point of view it's just a name.